open_ce/utils.py

```diff
@@ -295,3 +295,19 @@ def git_clone(git_url, git_tag, location):
         raise OpenCEError(Error.CLONE_REPO, git_url)
 
     return clone_successful
+
+def detect_mkl_feature():
```

Comment:
It would probably be better to have a `--use_mkl` argument to the build tools, instead of relying on environment variables to be set by the user.

Reply:
You think so? I think tool level arguments should be reserved for options that are more or less universal across the feedsocks. Other feedstocks could have some MKL specific function, but not many (maybe only PyTorch).

Reply:
I and Jason had discussed this over slack if we want to add environment variable or an argument to the build tools. 
@jayfurmanek and @bnemanich - Please let me know if we want to add an argument too.

Reply:
If we are already adding logic in the tools to handle this specific environment variable, it would be better to use an argument. Another option would be to set whether to use MKL as part of the conda_build_config.yaml file. I think this might actually be the preferable way.

Turning things on and off with environment variables can make things harder to reproduce when running within multiple environments.

Reply:
I think adding the flag in conda_build_config.yaml of tensorflow-feedstock is appropriate. I've implemented it and tested, it went fine. With this, tools won't need any change at all. Just the frameworks which have some MKL specific functions would need to make the needed change in their conda_build_config.yaml and the respective build scripts.